pkg/pub_dartdoc/bin/pub_dartdoc.dart

```diff
@@ -3,40 +3,23 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:dartdoc/dartdoc.dart';
-import 'package:dartdoc/src/dartdoc_options.dart';
-import 'package:dartdoc/src/logging.dart';
-import 'package:dartdoc/src/package_meta.dart';
+import 'package:dartdoc/options.dart';
 
 import 'package:pub_dartdoc/pub_data_generator.dart';
 
 void main(List<String> arguments) async {
-  final optionSet = await DartdocOptionSet.fromOptionGenerators(
-    'pub_dartdoc',
-    [
-      createDartdocOptions,
-      createLoggingOptions,
```

Comment:
Not sure exactly what these loggingoptions were, but it seems they are missing now.